lib/linguist/languages.yml

```diff
@@ -3879,6 +3878,17 @@ Lua:
   interpreters:
   - lua
   language_id: 213
+Luau:
+  type: programming
+  tm_scope: source.luau
+  ace_mode: lua
+  codemirror_mode: lua
+  codemirror_mime_type: text/x-lua
+  color: "#14a5ff"
+  extensions:
+  - ".luau"
+  interpreters:
+  - lua
```

Comment:
You've missed the step for adding the language ID. Also, this is used for scripts… is it common practice have shell scripts to be written in Luau?

Reply:
For whatever reason, none of the scripts were running on my system, and even after going through the hoops of installing Ruby. Would it be possible to determine a languageId manually?

As for the samples, I'll be working on adding more from public packages with real-world examples, apologies for not doing that sooner.

Reply:
> For whatever reason, none of the scripts were running on my system, and even after going through the hoops of installing Ruby. 

You could or [preferred method and use a Codespace](https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#using-github-codespaces) - everyone gets some free usage.

> Would it be possible to determine a languageId manually?

You can run [this line](https://github.com/github-linguist/linguist/blob/7ca3799b8b5f1acde1dd7a8dfb7ae849d3dfb4cd/script/update-ids#L9) manually in irb to get the ID.

Reply:
> > For whatever reason, none of the scripts were running on my system, and even after going through the hoops of installing Ruby.
> 
> You could use our [preferred method and use a Codespace](https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#using-github-codespaces) - everyone gets some free usage.

Alright, I'll give a shot with Codespace, thank you!

> > Would it be possible to determine a languageId manually?
> 
> You can run [this line](https://github.com/github-linguist/linguist/blob/7ca3799b8b5f1acde1dd7a8dfb7ae849d3dfb4cd/script/update-ids#L9) manually in irb to get the ID.

IRB doesn't want to run that, so I'll stick to trying on a Codespace.

Reply:
Resolved [this languageId issue here](https://github.com/github-linguist/linguist/pull/6612/commits/ba342d6f93708df835cd9e78f7f91d6e118e71cb).